bin/run-test.sh

```diff
@@ -39,7 +39,11 @@ function run {
 }
 
 # Run test/coverage
-run test test/hooks
+run `find test -maxdepth 1 -name '*.js' -not -name test-trace-agent.js`
+run test/test-trace-agent.js
+
+run test/hooks/*.js
```

Comment:
Let's see if we can revert this before landing. I linked a patch that should take care of this last test.